src/jobs/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import Layout from 'src/layout';
+import Head from 'next/head';
+
+import style from './index.module.css';
+import { useReducedMotion } from 'framer-motion';
+import { NextPage, InferGetStaticPropsType } from 'next';
+import { getStaticProps } from 'pages/jobs';
+import { ButtonNextLink } from 'src/components/button';
+import { BaseBlob } from '@variant/components/lib/blob';
+import { colors } from '@variant/profile';
+import { and } from 'src/utils/css';
+
+const JobsIndex: NextPage<InferGetStaticPropsType<typeof getStaticProps>> = ({
+  listings,
+}) => {
+  const reducedMotion = useReducedMotion();
+  return (
+    <>
+      <Layout>
+        <Head>
+          <meta
+            property="og:description"
+            content="En ny variant av et konsulentselskap som er raust, åpent og modig. Vi bistår med utvikling, design, salg, kursing og rådgiving."
+          />
+          <meta
+            name="description"
+            content="En ny variant av et konsulentselskap som er raust, åpent og modig. Vi bistår med utvikling, design, salg, kursing og rådgiving."
+          />
+          <title>Variant - Ledige stillinger</title>
+        </Head>
+        <section className={style.omVariant}>
+          <h2
+            className={style.omVariant__title}
+            data-no-animation={reducedMotion}
+          >
+            Hva står vi for?
+          </h2>
+          <article className={style.omVariant__wrapper}>
+            <div className={style.omVariant__blob}>
+              {/* TODO FIX WROK-AROUND when 2.0.7 is published  */}
```

Comment:
Is this comment referring to @variant/components, which now is set to 2.0.11 in `package.json`?

Reply:
Don't know what this is, so I'll just remove it 😅